src/ResourceManager/Network/Commands.Network/ExpressRouteCrossConnection/CrossConnectionPeering/RemoveAzureRMExpressRouteCrossConnectionPeeringCommand.cs

```diff
@@ -0,0 +1,102 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using MNM = Microsoft.Azure.Management.Network.Models;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.Remove, "AzureRMExpressRouteCrossConnectionPeering"), OutputType(typeof(PSExpressRouteCrossConnection))]
+    public class RemoveAzureRMExpressRouteCrossConnectionPeeringCommand : NetworkBaseCmdlet
+    {
+        public const string IPv4 = "IPv4";
```

Comment:
We should look at moving these definitions to [NetworkBaseCmdlet](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network/Common/NetworkBaseCmdlet.cs)

I see that its defined/used at least in one more place